app/views/questions/_active_questions.html.erb

```diff
@@ -2,7 +2,7 @@
 <% if @active_questions.empty? %>
   <p><em>There are no active questions at this time.</em></p>
 <% else %>
-    <table border="1">
+    <table class="striped responsive-table">
```

Comment:
Might not go into this PR and it's something we can talk about but the small screen version of the table might need to be tweaked.